packages/flutter/lib/src/rendering/mouse_tracking.dart

```diff
@@ -112,6 +116,7 @@ class MouseTrackerAnnotation with Diagnosticable {
       ifEmpty: '<none>',
     ));
     properties.add(DiagnosticsProperty<MouseCursor>('cursor', cursor, defaultValue: MouseCursor.defer));
+    properties.add(DiagnosticsProperty<bool>('valid', valid, defaultValue: true));
```

Comment:
Make it a `FlagProperty` and `ifFalse: 'invalid for MouseTracker'`

Reply:
Actually let's not add it here, since `MouseTrackerAnnotation` always return `valid: true`. Classes with a mutable `valid` will always have to _implement_ `MouseTrackerAnnotation` and have to reimplement this method. Add this line to `RenderMouseRegion` instead.